.travis.yml

```diff
@@ -0,0 +1,40 @@
+sudo: required
+dist: trusty
+language: node_js
+
+# test with node 4.*
+node_js:
+- "4"
```

Comment:
yes, please
we should be sure that we can switch to newer LTS asap
when it is ready